### PR TITLE
Adjust Info Screen to announce new Release-Features for 2.12 (EXPOSUREAPP-9499)

### PR DESCRIPTION
**Pull request by jurajkusnier**
image

## Jira Link
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9499

### CODE REVIEW

**Review by ChristinaTsigganos — APPROVED** [review]
UA approved

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml [hunk 1]
```diff
@@ -17,35 +17,37 @@
...
-• PCR-Tests
-• Schnelltests
-• Event Check-ins
-• Zertifikate"</item>
-        <item>"Sie können nun auch QR-Codes von Fotos oder PDF-Dateien über den QR-Code-Scanner importieren."</item>
-        <item>"Die minimale Kontaktdauer im Kontakttagebuch wurde von 15 auf 10 Minuten reduziert. Existierende Einträge zu Kontakten mit Personen wurden automatisch angepasst. Überprüfen und passen Sie die Einträge gegebenenfalls noch einmal an."</item>
+• 7-Tage-Inzidenz Hospitalisierung
+• COVID-19-Erkrankte auf Intensivstationen"</item>
+        <item>"Einstellungen für Mitteilungen werden ab jetzt zentral in den Einstellungen Ihres Smartphones verwaltet. Überprüfen Sie Ihre Mitteilungseinstellungen für die Corona-Warn-App in den Geräteeinstellungen erneut."</item>
+        <item>"Sie haben nun von der unteren Navigationsleiste aus direkten Zugriff auf den universellen QR-Code-Scanner."</item>
```
[the remark this answers is not in the document]
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:33]: The English word for it is "bottom navigation". Referring to this element: https://material.io/components/bottom-navigation

[the remark this answers is not in the document]
Comment (Ein-Tim) [reply Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:33]: I guess technically the text is alright but the I think normal users will wonder if there is a "upper navigation". I guess the UA should decide on this. CC @ChristinaTsigganos

**Review by marcauberer — APPROVED** [review]
lgtm

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ChristinaTsigganos) [reply Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:33]: Thanks for your feedback, @Ein-Tim! We'll look into this and get back to you soon. Best regards, the UA team

**Review by axelherbstreith — APPROVED** [review]

File: Corona-Warn-App/src/main/res/values-de/release_info_strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ChristinaTsigganos) [reply Corona-Warn-App/src/main/res/values-de/release_info_strings.xml:33]: Hi @Ein-Tim , we decided to leave the wording "untere Navigationsleiste" as is, as this was an important point in our team. However, thanks a lot for you feedback and have a great weekend!

**Pull request merged**